src/main/java/edu/harvard/iq/dataverse/engine/command/impl/CreateDataverseCommand.java

```diff
@@ -168,17 +169,8 @@ public Dataverse execute(CommandContext ctxt) throws CommandException {
     }
     
     @Override
-    public boolean onSuccess(CommandContext ctxt, Object r) {
-        try{
-            ctxt.index().indexDataverse((Dataverse) r);
-        } catch (IOException | SolrServerException e){           
-            Dataverse dv = (Dataverse) r;            
-            String failureLogText = "Indexing failed. You can kickoff a re-index of this dataverse with: \r\n curl http://localhost:8080/api/admin/index/dataverses/" + dv.getId().toString();
-            failureLogText += "\r\n" + e.getLocalizedMessage();
-            LoggingUtil.writeOnSuccessFailureLog(this, failureLogText,  dv);
-            return false;
-        }
-        return true;
+    public boolean onSuccess(CommandContext ctxt, Object r) {  
+        return ctxt.dataverses().index((Dataverse) r);
```

Comment:
The index dataverse here does not index permissions. Should you be using index((Dataverse) r, true)?

Reply:
Actually, the index without the boolean (of false) does index the call indePermissionForOneDVObject. 

indexPermissions or (index with true) adds a call to indexPermissionOnSelfAndChildren, but that shouldn't be needed in the case of creating a dataverse.